package.json

```diff
@@ -46,7 +46,8 @@
         "test": "turbo test",
         "test:unit": "turbo test:unit -- --silent",
         "test:react": "vitest --project @river-build/react-sdk",
-        "test:build": "turbo test:build"
+        "test:build": "turbo test:build",
+        "make-config": "node ./scripts/make-config.js"
```

Comment:
this is in the build because we include the file in the web3 package. it's not the best, the localhost addresses _should_ get pushed in via .env files, but that's not how it works now.